patches/minecraft/net/minecraft/client/gui/screens/worldselection/PresetEditor.java.patch

```diff
@@ -0,0 +1,10 @@
+--- a/net/minecraft/client/gui/screens/worldselection/PresetEditor.java
++++ b/net/minecraft/client/gui/screens/worldselection/PresetEditor.java
+@@ -28,6 +_,7 @@
+ 
+ @OnlyIn(Dist.CLIENT)
+ public interface PresetEditor {
++   @Deprecated /** @deprecated see {@link net.minecraftforge.client.PresetEditorManager} */
```

Comment:
Just a note: Patches do not have to be minimized, as long as the # of hunk lines is not changed. Merging these on the same line is not necessary. Compare this to, for example, what we do in [`BuiltinRegistries.java.patch`](https://github.com/MinecraftForge/MinecraftForge/blob/44c689f1712fcfb7ccb13262d1c02f0a880055be/patches/minecraft/net/minecraft/core/registries/BuiltInRegistries.java.patch#L8-L11). Having a `Forge: ` prefix is preferred for the case where mapping providers document the field alongside Forge, and then it is good to know which is which or where something came from (even if it is obvious in the case).